app_dart/lib/src/service/scheduler.dart

```diff
@@ -1588,7 +1590,8 @@ $stacktrace
                 log.warning(
                   'Asked to reschedule presubmits for unknown sha/PR: ${checkRunEvent.checkRun!.headSha!}',
                 );
-                return true;
+                // TODO(matanlurey): Follow up with jtmcdole on why this was/is a success case.
```

Comment:
follow up: from first glance; I have no clue. *maybe* related to "ack: please don't retry"

Reply:
Should this be a 4XX (user error) class?

Reply:
I think it's safe to make it a user error. WE should _not_ get unknown sha / PRs on cocoon.